cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -549,6 +549,20 @@ libxml_disable_entity_loader(true);
 
 A description of how to abuse this in PHP is presented in a good [SensePost article](https://www.sensepost.com/blog/2014/revisting-xxe-and-abusing-protocols/) describing a cool PHP based XXE vulnerability that was fixed in Facebook.
 
+# Python
+The following information for XXE injection in python is directly from this [official python documentation page](https://docs.python.org/2/library/xml.html#xml-vulnerabilities).
+
+The below table gives an overview of various modules in python used for xml parsing and whether they are vulnearble or not.
+| kind                      | sax        | etree      | minidom    | pulldom    | xmlrpc     |
```

Comment:
This table is broken if you look at the rendered markdown - I think need a blank line before it to render properly. Also, would something like `Attack` or `Attack Type` read better than `kind`?

Reply:
I've fixed the table now.